anni/src/database/helpers/cpic-constructors.ts

```diff
@@ -6,14 +6,20 @@ function guidelineFromRecommendation(
     recommendation: CpicRecommendation,
     source: string,
 ): IGuideline_Any {
+    function makeStringValuesLists(object: { [key: string]: string }): {
+        [key: string]: [string];
+    } {
+        const emptyNewList = new Object() as { [key: string]: [string] };
+        return object != undefined
+            ? Object.entries(object).reduce((reducedObject, [key, value]) => {
+                  reducedObject[key] = [value];
+                  return reducedObject;
+              }, emptyNewList)
+            : emptyNewList;
+    }
     return {
-        lookupkey: Object.entries(recommendation.lookupkey).reduce(
-            (lookupkey, [gene, phenotype]) => {
-                lookupkey[gene] = [phenotype];
-                return lookupkey;
-            },
-            new Object() as { [key: string]: [string] },
-        ),
+        lookupkey: makeStringValuesLists(recommendation.lookupkey),
+        phenotypes: makeStringValuesLists(recommendation.phenotypes),
```

Comment:
Here I thought it might be reasonable to use the gene's lookupkey value as a fallback if no phenotype is given so we can guarantee we always have a phenotype. This might make creating the `guidelineDescription` in `src/database/helpers/guideline-data.ts` easier.

Reply:
Good point, I will add this! 🙂